packages/atlas/src/views/global/NftSaleBottomDrawer/NftForm/NftForm.tsx

```diff
@@ -93,7 +93,7 @@ export const NftForm: FC<NftFormProps> = ({ setFormStatus, onSubmit, videoId })
     formState: { isValid },
   } = formMethods
 
-  const { video, loading: loadingVideo } = useFullVideo(videoId, { fetchPolicy: 'cache-only' })
+  const { video, loading: loadingVideo } = useFullVideo(videoId, { fetchPolicy: 'cache-first' })
```

Comment:
We could omit his option. `cache-first` is the default cache policy https://www.apollographql.com/docs/react/data/queries/#setting-a-fetch-policy

Reply:
The question is - do we need a full video here? Maybe this component could work with just a basic video fragments? If yes, then we can even keep the current `fetchPolicy` as the basic video will already be in the cache